docs/user_guide/1 - Installing.ipynb

```diff
@@ -30,7 +30,14 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "To additionally install the assignment toolbar notebook extension:"
+    "To additionally install, activate or deactivate the assignment toolbar notebook extension:"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To get help and see all the options you can pass while installing/activating nbgrader, use:"
```

Comment:
I would actually put the `python -m nbgrader --install --activate <profilename>` example first, then the `--help` example, since more people are going to just want to know how to install it and don't care about the other options.